pkg/gofr/datasource/solr/solr_test.go

```diff
@@ -177,9 +177,9 @@ func Test_ClientUpdateField(t *testing.T) {
 func Test_ClientDeleteField(t *testing.T) {
 	s := setupClient(t)
 
-	body := bytes.NewBuffer([]byte(`{"delete-field":{
+	body := bytes.NewBufferString(`{"delete-field":{
```

Comment:
Did you use mirror linter to detect these?

Reply:
Yes, I did check that each such instance is changed according to the linter.